apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryInsertResults.cs

```diff
@@ -135,14 +136,17 @@ public BigQueryInsertResults ThrowOnAnyError()
                 return this;
```

Comment:
Please could you fix the typo of "excetpion" on line 129 while you're here?

Reply:
Done